src/foam/nanos/auth/resetPassword/ResetPasswordTokenService.js

```diff
@@ -132,10 +132,8 @@ if ( ! Password.isValid(newPassword) ) {
 
 // update user's password
 userResult = (User) userResult.fclone();
-userResult.setPasswordLastModified(Calendar.getInstance().getTime());
```

Comment:
Reason this and next line are deleted?

Reply:
please see this [code](https://github.com/foam-framework/foam2/blob/master/src/foam/nanos/auth/UserPasswordHashingDAO.java#L31), as long as the DesiredPassword is not empty, it will update the LastModified and previousPassword there.